packages/common/src/eips.ts

```diff
@@ -471,4 +471,17 @@ export const EIPs: EIPsDict = {
     minimumHardfork: Hardfork.London,
     requiredEIPs: [],
   },
+  7516: {
+    comment: 'BLOBBASEFEE opcode',
+    url: 'https://eips.ethereum.org/EIPS/eip-7516',
+    status: Status.Draft,
+    minimumHardfork: Hardfork.London,
```

Comment:
London is too low, should be Shanghai

Reply:
updated min hf to `Cancun` as blobs will be coming `Cancun` onwards

Reply:
Ah, no, that's always the same discussion: the definition of this is "the minimum HF where this EIP (+ eventually other EIPs which are necessary for this one, so this would be 4844, but this part is covered by the EIP dependency setting) can be run in isolation". So this would be Shanghai.

Reply:
updated to Paris post discussion to confirm with `4844` settings :+1: